shedlock-core/src/main/java/net/javacrumbs/shedlock/core/LockConfiguration.java

```diff
@@ -36,18 +37,29 @@
      */
     private final Instant lockAtLeastUntil;
 
+    private final Clock clock;
+
     public LockConfiguration(@NotNull String name, @NotNull Instant lockAtMostUntil) {
-        this(name, lockAtMostUntil, Instant.now());
+        this(name, lockAtMostUntil, Instant.now(), Clock.systemUTC());
+    }
+
+    public LockConfiguration(@NotNull String name, @NotNull Instant lockAtMostUntil, @NotNull Clock clock) {
```

Comment:
Is this constructor necessary? If not, I would only add the on on line 54.

Reply:
It was added to support similar cases that LockConfiguration already offered, otherwise unnecessary, yes.